.env.development

```diff
@@ -1,3 +1,4 @@
 VITE_API_URL=http://localhost:9999
 VITE_CONTEXT=''
-VITE_ADMIN_REGISTRATION_ONLY=false
\ No newline at end of file
+VITE_ADMIN_REGISTRATION_ONLY=false
```

Comment:
In some other PR we should change it to something like FTA_FMEA_ADMIN_REGISTRATION_ONLY right? 

It was described in https://github.com/kbss-cvut/record-manager-ui/issues/88#issuecomment-1957356108.